libs/ui/input/helm/src/lib/hlm-input.directive.ts

```diff
@@ -4,7 +4,7 @@ import { cva, VariantProps } from 'class-variance-authority';
 import { ClassValue } from 'clsx';
 
 export const inputVariants = cva(
-	'flex rounded-md border font-normal border-input bg-transparent text-sm ring-offset-background file:border-0 file:text-foreground file:bg-transparent file:text-sm file:font-medium placeholder:text-muted-foreground focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:cursor-not-allowed disabled:opacity-50',
+	'flex rounded-md border font-normal border-input bg-transparent text-sm ring-offset-background file:border-0 file:text-foreground file:bg-transparent file:text-sm file:font-medium placeholder:text-muted-foreground focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-ring  disabled:cursor-not-allowed disabled:opacity-50',
```

Comment:
I'm curious about the reason behind this change?

Reply:
when focused, it was adding a "double" ring around the input, which looks really bad when it has an error, or even when focused without an error

Reply:
Can you send me a screenshot of what you see? I am wondering if this is just how it looks with shadcn or if there's something wrong. Thank you!